doc/site/changelogs/running-changelog.markdown

```diff
@@ -50,6 +50,7 @@ Expect a rescan of archives.
 * the default value for the `TooltipGeometry` springsetting has the Y coordinate moved from 0 to 0.125.
 * server no longer automatically forcestarts the game if there is nobody connected after 30s.
 * default tooltip now has income and harvest storage for each resource in its own line.
+* added Lua Language Server support: Files in `rts/Lua` are documented using LLS compatible annotations. Definitions are exported automatically to a Lua library repo for games that use the engine. Also updated Lua API doc generation to use LLS, with a regression in quality.
```

Comment:
Let's make it it's own section, it's not as minor a change as others here.

Note that the way to document the engine has also changed now, referencing `doc/site/guides/lua-...`.

`s/with a regression in quality/with a regression in quality, but permitting progressive enhancements in the future/` - or something like it.

Reply:
Oops, I didn't scroll down very far. Thought the top section was it.

See: 9237611eee5de290ff35025553c837194a684848

Reply:
Resolve if you're satisfied with that entry. Happy to make changes.